docs/developer/reference/standards.rst

```diff
@@ -0,0 +1,64 @@
+Standards
+=========
+
+This document defines the code and documentation standards used in this
+repository.
+
+Code Standards
+--------------
+
+The code in this repository conforms to standards set by the following tools:
+
+- black_ for code formatting
+- flake8_ for style checks
+- isort_ for import ordering
+- mypy_ for static type checking
+
+.. seealso::
+
+    How-to guides `../how-to/lint` and `../how-to/static-analysis`
+
+.. _documentation_standards:
+
+Documentation Standards
+-----------------------
+
+Docstrings are pre-processed using the Sphinx Napoleon extension. As such,
+google-style_ is considered as standard for this repository. Please use type
+hints in the function signature for types. For example:
```

Comment:
The style we used to use for this repo was https://numpydoc.readthedocs.io/en/latest/format.html. I am OK with the google one though, just needs some work on updating all docstrings.

Reply:
I noticed the numpy was used, let's update the rst in the skeleton to match the code

Reply:
I would say we should keep it in the numpydoc style

Reply:
Thank you!